### PR TITLE
[gis] Added gis module + websockets

### PR DESCRIPTION
**Pull request by nemesisdesign**
@moonwolf-github could you tell me if the name of the packages for RedHat/Fedora are correct?

### CODE REVIEW

File: tasks/yum.yml [hunk 1]
```diff
@@ -9,18 +9,29 @@
...
     - redhat-rpm-config
 
+- name: Install spatialite
+  when: openwisp2_database.engine == "django.contrib.gis.db.backends.spatialite"
+  apt: name={{ item }} state=latest
+  notify: reload systemd
+  with_items:
+    - sqlite
+    - gdal
+    - libproj-devel
```
Comment (moonwolf-github) [inline tasks/yum.yml]: Didn't find this. But found `libprojectM-devel`, not sure is this the same?

File: tasks/yum.yml [hunk 2]
```diff
@@ -9,18 +9,29 @@
...
+    - libgeos-dev
```
Comment (moonwolf-github) [inline tasks/yum.yml]: Didn't find this either. I'm not sure what to looking for.

**Comment by moonwolf-github** [comment]
As of `libgeos`:
```
# yum search geos
geos.x86_64 : GEOS is a C++ port of the Java Topology Suite
geos-debuginfo.x86_64 : Debug information for package geos
geos-devel.x86_64 : Development files for GEOS
geos-php.x86_64 : PHP modules for GEOS
geos-python.x86_64 : Python modules for GEOS
spatial4j.noarch : A Geospatial Library for Java
```

File: tasks/yum.yml (hunk 2 above)
Comment (nemesisdesign) [reply tasks/yum.yml]: may be `geos-devel`?
https://stackoverflow.com/questions/42097501/centos-libgeos-repository-missing

File: tasks/yum.yml (hunk 1 above)
Comment (nemesisdesign) [reply tasks/yum.yml]: Found this while googling: http://rpm.pbone.net/index.php3/stat/4/idpl/29688291/dir/centos_6/com/libproj-devel-4.8.0-24.1.x86_64.rpm.html
Isn't it available on the system you use?

Here's another one named `proj-devel`:
https://centos.pkgs.org/7/epel-x86_64/proj-devel-4.8.0-4.el7.x86_64.rpm.html

Comment (moonwolf-github) [reply tasks/yum.yml]: I was looking only for CentOS7. And yes, `proj-devel` is available indeed for CentOS7 and CentOS6.

File: tasks/yum.yml (hunk 2 above)
Comment (moonwolf-github) [reply tasks/yum.yml]: May be (i found `geos` in EPEL package but i'm not 100% sure it is the same)

Comment (nemesisdesign) [reply tasks/yum.yml]: usually devel packages contain shared libraries that are used for more advanced stuff, the stuff this kind of application usually need

**Comment by nemesisdesign** [comment]
@moonwolf-github thanks for helping out. I added a [task on the Google Code-In system to ask a student to help us test this branch](https://codein.withgoogle.com/tasks/6132543835340800/?sp-organization=5693741010518016), if you would like to mentor the student that will apply for this task I will be happy to have you on board!

**Comment by moonwolf-github** [comment]
I'll be happy to help but note that i never done that before. What am i suppose to do?

**Comment by nemesisdesign** [comment]
@moonwolf-github I'll send you an email ;-)

**Comment by nemesisdesign** [comment]
@moonwolf-github one of our GCI students, @R9295 tried the patch on a Centos VM and found out an issue which I resolved, I will merge the PR and move on to release next week after #49 is fixed.

**Pull request merged**